src/services/YotiService.ts

```diff
@@ -0,0 +1,341 @@
+/* eslint-disable no-console */
+import { Logger } from "@aws-lambda-powertools/logger";
+import crypto from "crypto";
+import { v4 as uuidv4 } from "uuid";
```

Comment:
can we use crypto randomUUID here pls, [seems to be faster than uuid](https://medium.com/@matynelawani/uuid-vs-crypto-randomuuid-vs-nanoid-313e18144d8c)